std_msgs/Header header
ssd_detection/SSD_Object[] objects
